Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Build the operator binary
+FROM golang:1.12.5 as builder
+
+WORKDIR /workspace
+
+# copy build manifests
+COPY Makefile Makefile
+
+# copy go modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+
+# ensure vendoring is up-to-date
+# cache deps before building and copying source so that we don't need to 
+# re-download as much and so that source changes don't invalidate our 
+# downloaded layer
+RUN make vendor
+
+# copy go source code
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+
+# build the binary
+RUN make lbins
```

Comment:
Good that we are building this inside container, we no longer have to depend on the installation of updated version of go locally, only image change is required.